source/MRViewer/MRRibbonSceneObjectsListDrawer.cpp

```diff
@@ -74,6 +74,7 @@ void RibbonSceneObjectsListDrawer::drawCustomObjectPrefixInScene_( const Object&
             fontManager.getFontSizeByType( RibbonFontManager::FontType::Icons );
         ImGui::PushFont( font );
 
+        ImGui::SetCursorPosY( ImGui::GetCursorPosY() + ( imageSize - ImGui::GetFontSize() ) * 0.5f );
```

Comment:
What's the purpose?

Reply:
As far as we do not provide icons python module uses symbol from font, it was bug here and this font symbol was drawn in wrong place (we just have not seen this because in application we use icons)